tests/ops/test_qubit_ops.py

```diff
@@ -1046,6 +1046,42 @@ def test_init_incorrect_pauli_word_length_error(self, pauli_word, wires):
         ):
             qml.PauliRot(0.3, pauli_word, wires=wires)
 
+    @pytest.mark.parametrize(
+        "pauli_word",
+        [
+            ("XIZ"),
```

Comment:
```suggestion
            ("XIZ"),
            ("IIII"),
```
This should cover the case where only I's are used.

Reply:
cool

Reply:
Thanks!